Makefile

```
@@ -0,0 +1,22 @@
+CHART_NAME ?= cray-vpa
+CHART_PATH ?= kubernetes
+CHART_VERSION ?= local
+
+HELM_UNITTEST_IMAGE ?= quintush/helm-unittest:3.3.0-0.2.5
+
+
+all : charts
+charts: chart_setup chart_packages chart_tests
+chart_packages: chart_package chart_package_2
```

This is failing the build.
```suggestion
charts: chart_setup chart_package chart_tests
```